DuckDB.NET.Test/TableFunctionTests.cs

```diff
@@ -174,4 +174,34 @@ public void RegisterTableFunctionWithBigInteger()
 
         data.Should().BeEquivalentTo(BigInteger.Parse("123456789876543210").ToByteArray().Select(b => TimeSpan.FromDays(1 + b)));
     }
+
+    [Fact]
+    public void RegisterTableFunctionWithErrors() {
+        Connection.RegisterTableFunction<string>("bind_err", parameters => {
+            throw new Exception("bind_err_msg");
+        }, (item, writer, rowIndex) => {
+        });
+
+        Assert.Throws<DuckDBException>(() => {
+            try {
+                var data = Connection.Query<int>($"SELECT * FROM bind_err('')").ToList();
+            } catch (Exception ex) {
+                Assert.Contains("bind_err_msg", ex.Message);
```

Comment:
`Assert.Throws` returns the exception that was caught so you can use that for checking message instead of putting try-catch here